src/test/testRunner.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as legacyTestRunner from '../../third-party/test/index'
+
+export interface RunTestsParameters {
+    /**
```

Comment:
nit: ditto

Reply:
I feel that the comments in this file will be more useful as more time passes between revisiting this code.

Reply:
if rootTestsPath is too ambiguous as time goes on a different name should be chosen